lib/Server.js

```diff
@@ -668,17 +671,9 @@ Server.prototype.checkHost = function (headers, headerToCheck) {
     false,
     true
   ).hostname;
-  // always allow requests with explicit IPv4 or IPv6-address.
-  // A note on IPv6 addresses:
-  // hostHeader will always contain the brackets denoting
-  // an IPv6-address in URLs,
-  // these are removed from the hostname in url.parse(),
-  // so we have the pure IPv6-address in hostname.
-  if (ip.isV4Format(hostname) || ip.isV6Format(hostname)) {
-    return true;
-  }
+
   // always allow localhost host, for convience
-  if (hostname === 'localhost') {
+  if (hostname === 'localhost' || hostname === '127.0.0.1') {
```

Comment:
If they use a local IP, developers should specify `public: '192.168.0.x'`. It will be covered by the `public` option further down in the code:

https://github.com/webpack/webpack-dev-server/blob/dd92f3120ff5f7a25c38833f88d490bef8f0c4e5/lib/Server.js#L714-L716

`allowedHosts` can also be used.